src/ui/pages/Overview/Positions/Positions.tsx

```diff
@@ -407,35 +407,39 @@ function ProtocolHeading({
   const { currency } = useCurrency();
 
   return (
-    <HStack gap={8} alignItems="center">
-      {dappInfo.id === DEFAULT_PROTOCOL_ID ? (
-        <WalletIcon />
-      ) : (
-        <TokenIcon
-          src={dappInfo.icon_url}
-          symbol={dappInfo.name || dappInfo.id}
-          size={24}
-          style={{ borderRadius: 6 }}
-        />
-      )}
-      <UIText kind="body/accent">
-        {dappInfo.name || dappInfo.id}
-        {' · '}
-        <NeutralDecimals parts={formatCurrencyToParts(value, 'en', currency)} />
-      </UIText>
-      <UIText
-        inline={true}
-        kind="caption/accent"
-        style={{
-          paddingBlock: 4,
-          paddingInline: 6,
-          backgroundColor: 'var(--neutral-200)',
-          borderRadius: 8,
-        }}
-      >
-        {`${formatPercent(relativeValue, 'en')}%`}
-      </UIText>
-    </HStack>
+    <VStack gap={8}>
```

Comment:
You wrapped one element in VStack which does nothing

Reply:
✅